site/content/en/docs/tasks/operations/standalone-deployment-mode.md

```diff
@@ -31,14 +31,14 @@ Create a local directory just for testing:
 mkdir -p /tmp/envoy-gateway-test
 ```
 
-As we do not provide the Envoy Gateway binary in latest release,
-you can compile this binary on your own from project by using command:
+You can download the Envoy Gateway binary from [Github release](https://github.com/envoyproxy/gateway/tags),
```

Comment:
I think something like `https://github.com/envoyproxy/gateway/releases/download/{{< yaml-version >}}/quickstart.yaml` will be super helpful.

Reply:
great idea! updated